docs/api/rebac/groups.mdx

```diff
@@ -6,64 +6,111 @@ title: Groups API
+:::info How it Works:
+Say we create a marketing group and assign a `member` role to it, we then assign users to the Group via a `member` role. The API then [derives](https://docs.permit.io/how-to/build-policies/rebac/overview/#role-derivation) other roles based on this assignment (See example below)
+:::
 
-### Assign users to a group:
+- **Resource Instances**: Grouping resources in a Resource Group enables you to manage access control for multiple resources simultaneously, ensuring all included resources share the same permission configuration.
 
-When you assign "user-1" to 'company' group, the following will by automated by API:
-- assign 'user-1' to 'company#member' role
+:::info How it Works:
+Following the previous example, we can create another group in our marketing group titled `social_media`. This group will contain resource instances of marketing materials. We can assign this newly created group the role of editor.
+The Groups API creates an automatic derivation between the `social_media#editor` role and the `marketing#member` role - meaning all users with the `marketing#member` role will automatically be assigned with the `social_media#editor` role. New resources added to `social_media` will be accessible to edit by members of the marketing group, and new users added as members to the marketing group will automatically have editor access to social media assets.
+:::
 
-and the user will also get the role 'marketing#editor' in the resource 'marketing#social_media'.
+## Example Usage for User Groups:
 
+Let’s dive deeper into a simple use-case example for User Groups:
 
-## Create a Group via API
+Say our company has several teams, each with a few employees assigned to each team. Our goal is to allow all members of the Marketing Team to Edit Social Media resources based on their group membership.
 
-To create a new group, make a POST request to the following with the required data:
+For this purpose, we will first create a group named `teams`. The group, `teams` is a resource, while each specific team (Such as `marketing-team`) is a resource instance.
+
+Users are assigned to the `marketing-team` via the `member` role (The `member` role here is, of course, just an example. Feel free to customize roles as you wish).
+
+As mentioned above, we also have a `social-media` resource, and a `training_video` - which is a resource instance of the `social-media` type. This resource instance supports the role `editor`.
+
+Let’s set this up:
+
+### API Calls: Create Group, Add Role to Group, Add Users to Group:
+
+
+To **create a new group**, make a POST request with the following data:
 
 ```bash
 curl 'https://api.permit.io/v2/facts/{project_id}/{env_id}/groups' \
-  -H 'authorization: Bearer API_SECRET_KEY' \
-  --data-raw '{"group_instance_key": "Acme"}'
+ -H 'authorization: Bearer API_SECRET_KEY'
+ --data-raw '{"group_instance_key": "marketing", “group_resource_type_key": "teams"}' \
```

Comment:
remove the `\` at the end of the line

Reply:
The double quotes are not fine - need to be " instead of “ - otherwise it's not working.